src/NUnitFramework/tests/Attributes/TestCaseSourceTests.cs

```diff
@@ -401,4 +402,124 @@ protected static IEnumerable<bool> InheritedStaticProperty
             get { yield return true; }
         }
     }
+
+    [TestFixture]
+    public class TestCaseSourceArrayBoxingWithASingleValue
+    {
+        public static int[][] TestCaseData = { new[] { 1 } };
```

Comment:
IIRC, this was not a case we previously permitted. Is there a reason we should expand to permit this? I'll discuss this more in the general comment.